content/en/news/spanish-traditional-chinese-available.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Please add keywords and description here, ref: https://github.com/kubesphere/website/blob/master/content/en/news/kubesphere-geko-partnership.md

Reply:
keywords and description added